pkg/controller/metrics/metrics.go

```diff
@@ -201,12 +232,41 @@ func (mc *Calculator) Start(ctx context.Context) error {
 	wait.UntilWithContext(ctx, func(ctx context.Context) {
 		mcLog := log.WithField("controller", "metrics")
 		recobsrv := NewReconcileObserver(ControllerName, mcLog)
+		mConfig, err := readMetricsConfig()
+		if err != nil {
+			log.WithError(err).Error("cannot read metrics config")
+			return
```

Comment:
Hm, this means someone could (accidentally or deliberately) disable this whole module with a bogus HiveConfig.

Looking at the code below, I *think* you can just remove this `return` and everything will still work. (Obviously in that scenario none of the metrics will be reported.)
```suggestion
```